src/cloudant/database.py

```diff
@@ -762,6 +762,48 @@ def view_cleanup(self):
 
         return resp.json()
 
+    def get_list_function_result(self, ddoc_id, list_name, view_name, **kwargs):
+        """
+        Retrieves a customized MapReduce view result from the specified
+        database based on the list function provided.  List functions are
+        used when you want to access Cloudant directly from a browser, and
```

Comment:
Whilst a browser is the most likely use case, I don't think we should imply it is the only use case, perhaps just add "for example" e.g. 
> ... used<em>, for example,</em> when you ...